pytest_test_groups/__init__.py

```diff
@@ -21,15 +22,21 @@ def pytest_addoption(parser):
                     help='The number of groups to split the tests into')
     group.addoption('--test-group', dest='test-group', type=int,
                     help='The group of tests that should be executed')
+    group.addoption('--random-seed', dest='random-seed', type=float,
```

Comment:
Do you think this should be namespaced like `--test-group-random-seed` to avoid confusion with *something* else that may be passing in a random seed?